notebook/static/notebook/js/codecell.js

```diff
@@ -171,6 +171,8 @@ define([
             if (that.keyboard_manager) {
                 that.keyboard_manager.enable();
             }
+
+            that.code_mirror.setOption('readOnly', that.is_readonly());
```

Comment:
Yeah, I've duplicated it in textcell.js, which seems to be the only other cell type defined? I can't really set this in the parent cell.js since no codemirror stuff happens there.